WrightTools/data/_data.py

```diff
@@ -1928,6 +1929,94 @@ def transform(self, *axes, verbose=True):
         elif verbose and newt and not nownewt:
             print("I got better")
 
+    def translate_to_txt(
+        self, filepath, delimiter="\t", channels=None, variables=None, fmt=".5g", verbose=True
+    ):
+        """
+        Write a serialized, readable list of the channels and variables to
+        file. Each line (row) denotes a seperate data point. Each column
+        represents a unique variable or channel. Axes are neglected.
+
+        Parameters
+        ----------
+
+        filepath: path-like
+            path to output file.
+        delimiter: str
+            separation character.  Defaults to "\t"
+        channels: list[str] (optional)
+            List of channel names to include. Default is all channels.
+        variables: list[str] (optional)
+            List of variable names to include. Default is all variables.
+        fmt: [python format spec](https://docs.python.org/3/library/string.html#formatspec)
+            format specifier for variables and channels
+
+        Returns
+        -------
+
+        None
+
+        Notes
+        -----
+
+        * This is a lossy write procedure; some properties, such as axes, are
+        not recorded.
+        * The shape structure of the data is recorded as a series of indices
+        (`{a_i}`) comprising the first few columns. A vertical line separates
+        these indexes from variables and channels
+        * wt5, the native file format for Data objects, is a specific variant
+        of the HDF5 file format. HDF5 has well-developed tools for working with
+        generic datasets.  See
+        https://portal.hdfgroup.org/display/knowledge/How+to+convert+an+HDF4+or+HDF5+file+to+ASCII+%28text%29+or+Excel
+        for more information.
+        """
+        # attrs
+        import tidy_headers
+
+        tidy_headers.write(filepath, {k: v for k, v in self.attrs.items()})
```

Comment:
```suggestion
        tidy_headers.write(filepath, self.attrs)
```

I think this should work